cpp/src/hash/concurrent_unordered_map.cuh

```diff
@@ -543,6 +543,6 @@ class concurrent_unordered_map {
         m_hashtbl_values, m_capacity, m_unused_key, m_unused_element);
     }
 
-    CUDA_TRY(cudaGetLastError());
+    CUDF_CUDA_TRY(cudaGetLastError());
```

Comment:
@davidwendt I think the calls to `CUDF_CUDA_TRY(cudaGetLastError()));` like this one should be replaced with `CUDF_CHECK_CUDA(stream);`.

The check macro would expand to `CUDF_CUDA_TRY(cudaPeekAtLastError());` in release builds. Does that seem right to you?

Reply:
Yes, that seems fine to me.